### PR TITLE
[ATIP] add step for getting device dumped xml

### PR DESCRIPTION
**Pull request by jacky-young**
-add step for getting device dumped xml content

### CODE REVIEW

**Comment by jacky-young** [comment]
tested locally passed, specially for nexus3. @Honry please help to review it.

File: tools/atip/README.md [hunk 1]
```diff
@@ -424,6 +424,7 @@ def i_visit_url(context, url):
 @step(u'I save process of finding view "{params_kw1}" on the "{position}" side of view "{params_kw2}"')
 @step(u'I reload above process and save result to object "{key}"')
 @step(u'I wait saved object "{key}" gone in {time_out:d} seconds')
+@step(u'I except the content "{text}" in the dumped xml')
```
Comment (Honry) [inline tools/atip/README.md]: 's/except/expect'

File: tools/atip/atip/android/steps.py [hunk 2]
```diff
@@ -299,3 +299,14 @@ def wait_object_gone(context, key, time_out):
 def open_notification(context, element_text, key):
     elecount = context.android.d(text=element_text).count
     assert context.android.save2InfoTemp(elecount, key)
+
+
+#get the dumped content(unicode) from return
+#check the excepted content in the dump
+@step(u'I excpet the content "{text}" in the dumped xml')
```
Comment (Honry) [inline tools/atip/atip/android/steps.py]: ditto

File: tools/atip/atip/android/steps.py [hunk 3]
```diff
@@ -299,3 +299,14 @@ def wait_object_gone(context, key, time_out):
 def open_notification(context, element_text, key):
     elecount = context.android.d(text=element_text).count
     assert context.android.save2InfoTemp(elecount, key)
+
+
+#get the dumped content(unicode) from return
+#check the excepted content in the dump
```
Comment (Honry) [inline tools/atip/atip/android/steps.py]: ditto

**Comment by Honry** [comment]
LGTM

**Pull request merged**